migration/migration.go

```diff
@@ -630,14 +625,12 @@ func createSpace(ctx context.Context, spaceRepo *space.GormRepository, id uuid.U
 
 func createOrUpdateWorkItemLinkType(ctx context.Context, linkCatRepo *link.GormWorkItemLinkCategoryRepository, linkTypeRepo *link.GormWorkItemLinkTypeRepository, spaceRepo *space.GormRepository, linkType *link.WorkItemLinkType) error {
 	existingLinkType, err := linkTypeRepo.Load(ctx, linkType.ID)
-	cause := errs.Cause(err)
-	switch cause.(type) {
-	case errors.NotFoundError:
+	if ok, _ := errors.IsNotFoundError(err); ok {
 		_, err := linkTypeRepo.Create(ctx, linkType)
 		if err != nil {
 			return errs.WithStack(err)
 		}
-	case nil:
+	} else {
 		log.Info(ctx, map[string]interface{}{
 			"wilt": linkType.Name,
 		}, "Work item link type %s exists, will update/overwrite all fields", linkType.Name)
```

Comment:
@kwk what if the error from load is `NewInternalError` ? This piece of code is missing this scenario at this moment.